components/support/error/Cargo.toml

```diff
@@ -6,7 +6,15 @@ edition = "2021"
 license = "MPL-2.0"
 
 [dependencies]
+log = "0.4"
+parking_lot = "0.12"
```

Comment:
Do you think it's worth having this be `">=0.11,<=0.12"` to prepare for a future where this ends up in m-c? (OTOH, I don't really case as we don't have concrete plans there yet, and that will likely cause other yaks)